packages/terra-application-header-layout/tests/jest/ApplicationHeader.test.jsx

```diff
@@ -1,121 +1,175 @@
 import React from 'react';
+import { IntlProvider } from 'terra-base';
 
 import ApplicationHeaderLayout from '../../src/ApplicationHeaderLayout';
 
 describe('ApplicationHeaderLayout', () => {
   it('should render without optional props', () => {
-    const result = shallow(<ApplicationHeaderLayout />);
+    const result = shallow(
+      <IntlProvider locale="en">
```

Comment:
We should be able to pass the intl context when rendering this component so we don't have to wrap it in the IntlProvider. Here is an example for reference:
- Setup Intl context: https://github.com/cerner/terra-core/blob/master/packages/terra-tabs/tests/jest/intl-context-setup.js
- Use in Jest test: https://github.com/cerner/terra-core/blob/master/packages/terra-tabs/tests/jest/Tabs.test.jsx#L8

Reply:
We actually aren't using the context for intl anymore, so I don't think this approach would help us. We did agree to using the IntlProvider on this code review. https://github.com/cerner/terra-core/pull/1956/files